core/src/main/resources/hudson/PluginManager/index.jelly

```diff
@@ -29,7 +29,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <local:table page="updates" list="${app.updateCenter.updates}" xmlns:local="/hudson/PluginManager">
     <div style="margin-top:1em">
-      ${%Select}: <a href="javascript:toggleCheckboxes(true);">${%All}</a>, <a href="javascript:toggleCheckboxes(false);">${%None}</a><br/>
+      ${%Select}: <a href="javascript:checkPluginsWithoutWarnings();">${%All}</a>, <a href="javascript:toggleCheckboxes(false);">${%None}</a><br/>
```

Comment:
Perhaps change label to *All compatible*? Or add a tool tip with that information?